camp/faq.html

```diff
@@ -22,7 +22,7 @@ <h2>Topics You Might Be Wondering About</h2>
   <h2>Upcoming Events</h2>
 
   <h3>When and where is the next Offline Camp?</h3>
-  <p>Offline Camp Oregon 2019 will take place in Grants Pass, Oregon from August 2-5, 2019.</p>
+  <p>Offline Camp Oregon 2019 will take place in Grants Pass, Oregon from September 27-30, 2019. (For the health and safety of our campers during the Milepost 97 wildfire, we've postponed the event, which previously schedule for August 2-5.)</p>
```

Comment:
"Which _was_ previously _scheduled_" (missing "was" and should be "scheduled" and not "schedule")